pkg/cmd/samples/list.go

```diff
@@ -27,13 +29,13 @@ func NewListCmd() *ListCmd {
 		Short: "List Stripe Samples supported by the CLI",
 		Long: `A list of available Stripe Sample integrations that can be setup and bootstrap by
 the CLI.`,
-		Run: listCmd.runListCmd,
+		RunE: listCmd.runListCmd,
 	}
 
 	return listCmd
 }
 
-func (lc *ListCmd) runListCmd(cmd *cobra.Command, args []string) {
+func (lc *ListCmd) runListCmd(cmd *cobra.Command, args []string) error {
```

Comment:
yes this one looks great, except previous comment about having to do `samples.Samples{...}` seems that it's an implementation detail to me (not 100% convinced tho). Especially since we only pass defaults to it...

It should probably just be `samples.GetSamples("list")` without having to init a sample